tensorflow_probability/python/experimental/bijectors/highway_flow_test.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2021 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Tests for HighwayFlow."""
+import tensorflow.compat.v2 as tf
+
+import tensorflow_probability as tfp
+from tensorflow_probability.python.internal import samplers
+from tensorflow_probability.python.internal import test_util
+
+tfb = tfp.bijectors
+tfd = tfp.distributions
+
+
+@test_util.test_all_tf_execution_regimes
+class HighwayFlowTests(test_util.TestCase):
+
+  def testBijector(self):
+    width = 1
+    for dim in range(2):
+      if dim == 0:
+        # Test generic case with scalar input
+        x = tf.ones((width,)) * samplers.uniform((width,), minval=-1.,
```

Comment:
Multiplying the samples by `tf.ones` here doesn't do anything; you can remove it (also in similar lines elsewhere).